gctf/protocols/protocol_gctf.py

```diff
@@ -251,7 +294,13 @@ def _estimateCTF(self, micFn, micDir, micName):
             pwutils.makePath(micDir)
             downFactor = self.ctfDownFactor.get()
             micFnMrc = self._getTmpPath(pwutils.replaceBaseExt(micFn, 'mrc'))
-            micFnCtf = self._getTmpPath(pwutils.replaceBaseExt(micFn, 'ctf'))
+
+            if gctf.Plugin.getActiveVersion() not in ['0.50', '1.06']:
```

Comment:
How would be possible put this in a function? The same condition is checked in many places in the code.